### PR TITLE
Support VideoFrame as a Texture resource.

### PR DESCRIPTION
**Pull request by orange4glace**
##### Description of change
Webcodecs API is now stable (even can be used on Safari).
Support [`VideoFrame`](https://developer.mozilla.org/en-US/docs/Web/API/VideoFrame) as a Texture resource.

##### Pre-Merge Checklist

- [ ] Tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] Lint process passed (`npm run lint`)
- [ ] Tests passed (`npm run test`)

### CODE REVIEW

**Comment by orange4glace** [comment]
Is there something more needed to be merged ? :]

**Comment by bigtimebuddy** [comment]
@orange4glace could you figure out why it is not passing Actions?

File: packages/core/src/textures/resources/VideoFrameResource.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { BaseImageResource } from './BaseImageResource';
+
+type VideoFrame = any;
```
Comment (SuperSodaSea) [inline packages/core/src/textures/resources/VideoFrameResource.ts:3]: Maybe we use [@types/dom-webcodecs](https://www.npmjs.com/package/@types/dom-webcodecs) for the `VideoFrame` type?

**Comment by SuperSodaSea** [comment]
Related: #9681

**Comment by orange4glace** [comment]
Turns out it was linter.
Although #9681 also enables VideoFrame either it is planned to distributed in v8 so I hope this would be merged and distributed in v7.4.0 if it can.

**Review by SuperSodaSea — APPROVED** [review]
Now `@types/dom-webcodecs@0.1.3` is used. I also did some document cleanup and it looks good to me now.

**Comment by orange4glace** [comment]
> Now `@types/dom-webcodecs@0.1.3` is used. I also did some document cleanup and it looks good to me now.

Thanks! 👍

**Comment by hlevring** [comment]
What is the estimated time frame for this to be landed in a public update to v7?

**Pull request merged**